src/EnergyPlus/DataSurfaces.cc

```diff
@@ -734,4 +734,34 @@ Real64 AbsBackSide(EnergyPlusData &state, int SurfNum)
     return AbsorptanceFromExteriorBackSide + AbsorptanceFromInteriorBackSide;
 }
 
+void GetVariableAbsorptanceSurfaceList(EnergyPlusData &state)
```

Comment:
If I am following correctly, this function will walk through every material layer of every surface even if there are zero `MaterialProperty:VariableAbsorptance` objects in the input. I would add a global flag `AnyVariableAbsorptance`, similar to `DataHeatBalance::AnyAirBoundary`, default to false, and set true in `GetVariableAbsorptanceInput`. Then this function can return immediately `!AnyVariableAbsorptance`.

Reply:
Indeed. I'll add a global flag to short-circuit it when no MaterialProperty:VariableAbsorptance is present.